cli/cmd/completion.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+var completionCmd = &cobra.Command{
+	Use:       "completion [bash|zsh]",
+	Short:     "Shell completion",
+	Long:      "Output completion code for the specified shell (bash or zsh).",
+	Args:      cobra.ExactArgs(1),
```

Comment:
😍 I didn't know about this. We should change other commands to use its variations.

Reply:
yeah it's a nice change. also available is `Args: cobra.OnlyValidArgs()`, which works in conjunction with `ValidArgs: []string{"bash", "zsh"}`.

but what I really want is `Args: cobra.ExactArgs(1) && cobra.OnlyValidArgs()`